packages/manager/modules/pci/src/projects/project/contacts/add/add.html

```diff
@@ -1,20 +1,21 @@
 <form novalidate>
     <oui-modal
-        data-title="{{:: 'cpb_rights_add_title' | translate}}"
-        data-primary-action="$ctrl.validateAddRight()"
-        data-primary-label="{{ ::'cpb_rights_add' | translate }}"
-        data-secondary-action="$ctrl.cancel()"
-        data-secondary-label="{{ ::'common_cancel' | translate }}"
-        data-on-dismiss="$ctrl.cancel()"
-        data-loading="$ctrl.loader">
+        data-title="{{ ::'cpb_rights_add_title' | translate }}"
+        data-primary-action="$ctrl.add()"
+        data-primary-label="{{ ::'cpb_rights_common_add' | translate }}"
+        data-secondary-action="$ctrl.goToContactsPage()"
+        data-secondary-label="{{ ::'cpb_rights_common_cancel' | translate }}"
+        data-on-dismiss="$ctrl.goToContactsPage()"
+        data-loading="$ctrl.isAdding">
 
-        <oui-field data-label="{{ :: $ctrl.model.isUSorCA ? 'cpb_rights_table_email' : 'cpb_rights_table_nichandle' | translate }}">
+        <oui-field data-label="{{ :: $ctrl.user.ovhSubsidiary === 'FR' ? 'cpb_rights_table_nichandle' : 'cpb_rights_table_email' | translate }}">
```

Comment:
warning i think you modify the behavior here, original test was : `indexOf(['US', 'CA'], me.country.toUpperCase())`

is it intentional ?

Reply:
yes, both are same, if not US and CA , its FR. Let me know if I am wrong.

Reply:
I think I am wrong here, let me change. Thanks for pointing this big mistake :)